src/integrations.ts

```diff
@@ -0,0 +1,28 @@
+import 'components/analytics'
```

Comment:
nit: this is probably out of scope for this PR, however, a nice follow-up would be to move `components/analytics` file out of there, since it's not a component, but a utility.